nsacyber/HIRS
# [#864] Undid Constant Name Rule In Checkstyles XML and Applied Formatting/Changes to HIRS_UTILS

### Description
I recently modified one of the modules in checkstyles.xml to ignore checks for Java constants that begin with an underscore. However, after further research and discussions with my colleagues, I believe we should revert this rule and instead update the Java constants to align with recommended Java guidelines. This change will help maintain code consistency and adherence to best practices. Apply the new changes to the entire HIRS_UTILS module.

### Other Changes You Might Have Noticed:
Asides from undoing one of the checkstyle modules in the checkstyles.xml file, I also "lombok-ed" a lot of java files in the UTILS files and a good portion of the files in the HIRS_UTILS module were formatted according to checkstyles guidelines. 

### Test Instructions:
1. Run `./gradlew clean HIRS_UTILS:check --rerun-tasks` in your terminal window to verify there are no checkstyle errors.
2. Run `./gradlew clean build --rerun-tasks` in your terminal window to verify there are no build errors.
3. Go to the SwidTagsConstants.java file and verify none of the constants start with an underscore.
